{
  "docs": "https://docs.ansible.com/ansible-core/2.15/dev_guide/testing/sanity/pylint.html",
  "results": [
    {
      "message": "The test `ansible-test sanity --test pylint` [[explain](https://docs.ansible.com/ansible-core/2.15/dev_guide/testing/sanity/pylint.html)] failed with 1 error:",
      "output": "tests/unit/compat/builtins.py:31:4: unused-import: Unused import __builtin__"
    }
  ],
  "verified": false
}
